src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/SealInternalTypesTests.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Xunit;
+
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.SealInternalTypes,
+    Microsoft.NetCore.Analyzers.Runtime.SealInternalTypesFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.SealInternalTypes,
+    Microsoft.NetCore.Analyzers.Runtime.SealInternalTypesFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class SealInternalTypesTests
```

Comment:
Can you tests for:

- Generic types (ie, a test that will fail if we remove `.OriginalDefinition` from the implementation)
- Partial classes (specifically for VB - sometimes `GetModifiers` API behavior is a bit unexpected depends on whether we pass the ClassBlock or ClassStatement node)